pandas/core/arrays/datetimelike.py

```diff
@@ -826,6 +826,9 @@ def searchsorted(self, value, side="left", sorter=None):
         indices : array of ints
             Array of insertion points with the same shape as `value`.
         """
+        if is_list_like(value):
```

Comment:
do we need to update the doc-string?

Reply:
I think it's okay since the docstring says value is "array_like" which I believe includes lists / tuples according to numpy documentation